src/Geta.Epi.FontThumbnail.Tests/Geta.Epi.FontThumbnail.Tests.csproj

```diff
@@ -5,6 +5,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="EPiServer.CMS.UI.Core" Version="11.1.0" />
```

Comment:
```suggestion
     <PackageReference Include="EPiServer.CMS.UI.Core" Version="11.2.1" />
```

Reply:
Aye. I just noticed that the Episerver packages in the main project are set to version 11.2.1 except for `EPiServer.CMS.UI.Core`. But shouldn't they all be set to the same version as the minimum version of the Episerver dependancy in the .nuspec file, 11.1.0 in this case?

Reply:
Committed a fix to this PR.